Library/ViewModels/PledgeViewModel.swift

```diff
@@ -26,9 +26,12 @@ public class PledgeViewModel: PledgeViewModelType, PledgeViewModelInputs, Pledge
       .map(first)
       .skipNil()
 
-    self.amountAndCurrency = projectAndReward.signal
+    self.amountAndCurrencyAndDeliveryDate = projectAndReward.signal
       .map { (project, reward) in
-        (reward.minimum, currencySymbol(forCountry: project.country).trimmed())
+        (reward.minimum, currencySymbol(forCountry: project.country).trimmed(),
+         reward.estimatedDeliveryOn.map {
+         Format.date(secondsInUTC: $0, template: "MMMMyyyy", timeZone: UTCTimeZone)
```

Comment:
Do you think it would make sense to return optional String? This way we don't have to provide default value of `""` and the view component should be able to handle `nil` as well.

Reply:
I think this is fine for now.